src/Container/CommandBusFactory.php

```diff
@@ -22,21 +23,20 @@
  */
 class CommandBusFactory extends AbstractBusFactory
 {
-
     /**
      * @inheritdoc
      */
-    protected function getBusClass()
+    public function containerId()
     {
-        return CommandBus::class;
+        return 'command_bus';
     }
 
     /**
      * @inheritdoc
      */
-    protected function getBusConfigKey()
+    protected function getBusClass()
```

Comment:
again, this is a service name registered in the container, it's not always a classname